extensions/binding/org.eclipse.smarthome.binding.homematic/src/main/java/org/eclipse/smarthome/binding/homematic/internal/communicator/AbstractHomematicGateway.java

```diff
@@ -131,6 +132,7 @@
         virtualDatapointHandlers.add(new DisplayTextVirtualDatapoint());
         virtualDatapointHandlers.add(new HmwIoModuleVirtualDatapointHandler());
         virtualDatapointHandlers.add(new PressVirtualDatapointHandler());
```

Comment:
What about this PressVirtualDatapointHandler? The implementation looks very similar to the new Handler, except that it uses the options "SHORT", "LONG", "LONG_RELEASE", "CONT" instead of SHORT_PRESSED, LONG_PRESSED, DOUBLE_PRESSED. It looks like the old implementation, but shouldn't this be obsolete now?

Reply:
Well, I think there are two drawbacks of PressVirtualDatapointHandler

1. It does not use the system channel, therefore it is specific to this binding.
2. The devices that I tested with, do not send PRESSED_LONG before you release the key. Therefore the events LONG and LONG_RELEASE appear almost at the same time on PressVirtualDatapoint

Therefore I aggree that ButtonVirtualDatapointHandler may deprecate PressVirtualDatapointHandler. However, this would be API breaking. That means people who use the current trigger (for instance in a rule), will loose compability if we remove it.

How about the following?

- I mark the class as deprecated (annotation)
- I update the binding's readme accordingly (maybe striketrough of PRESS channel and putting system.button channel instead)